addon/-private/drop-zone-state.js

```diff
@@ -44,11 +44,11 @@ export default EmberObject.extend({
 
   overlayText: computed('type', function() {
     let type = this.get('type');
-    return get(dropZone, `overlayText${ capitalize(type) }`);
+    return get(dropZone, `overlayText${capitalize(type)}`);
```

Comment:
Can (should?) we configure to keep these spaces?

Reply:
will look into this if possible